e2e/libs/metrics/metrics.py

```
@@ -0,0 +1,22 @@
+import time
+
+from kubernetes import client
+from kubernetes.client.rest import ApiException
+
+from utility.utility import get_retry_count_and_interval
+from utility.utility import logging
+
+def get_node_metrics(node_name, metrics_name):
+    retry_count, retry_interval = get_retry_count_and_interval()
+    for i in range(retry_count):
```

Ruff (0.8.2)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)